public/scripts/app.js

```diff
@@ -2,6 +2,94 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
 
-// your code
+// Profile GET Statement
+$.ajax({
+  method: "GET",
+  url:"/api/profile"
+})
```

Comment:
It's weird that you make this AJAX call without it doing anything; you could remove this call.